packages/mui-material/src/styles/CssVarsProvider.tsx

```diff
@@ -16,11 +16,10 @@ import { defaultConfig } from '../InitColorSchemeScript/InitColorSchemeScript';
 
 const defaultTheme = experimental_extendTheme();
 
-const {
-  CssVarsProvider,
-  useColorScheme,
-  getInitColorSchemeScript: deprecatedGetInitColorSchemeScript,
-} = createCssVarsProvider<SupportedColorScheme, typeof THEME_ID>({
+const { CssVarsProvider, useColorScheme, getInitColorSchemeScript } = createCssVarsProvider<
```

Comment:
We should add the `@deprecated` tag to `getInitColorSchemeScript` to hint at it in IDEs

Reply:
Also suggested here: https://github.com/mui/material-ui/pull/42247#discussion_r1664248112

Reply:
Added.